recipes/linkage2allegro/meta.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: linkage2allegro
+  version: "2017.1"
+
+source:
+  url: https://github.com/BioTools-Tek/linkage-converter/archive/master.zip
```

Comment:
Instead of `.../master.zip` can you provide a link to a release or an archive for a tag or commit, e.g.:
```yaml
  url: https://github.com/BioTools-Tek/linkage-converter/archive/d8eefb76ad2cf3ab5fb6b190e968f6f933cc2886.tar.gz
```